tenable/io/v3/vm/agents/api.py

```diff
@@ -0,0 +1,138 @@
+'''
+Agents
+======
+
+The following methods allow for interaction into the Tenable.io
+:devportal:`agents <agents>` API endpoints.
+
+Methods available on ``tio.v3.vm.agents``:
+
+.. rst-class:: hide-signature
+.. autoclass:: AgentsAPI
+    :members:
+'''
+from typing import Dict, Union
+from uuid import UUID
+
+from tenable.io.v3.base.endpoints.explore import ExploreBaseEndpoint
+from tenable.io.v3.vm.agents.schema import AgentsSchema
+
+
+class AgentsAPI(ExploreBaseEndpoint):
+    _path: str = 'api/v3/agents'
+    _conv_json: bool = True
+
+    def details(self, agent_id: UUID) -> Dict:
+        '''
+        Retrieves the details of an agent.
+
+        :devportal:`agents: get <agents-get>`
+
+        Args:
+            agent_id (UUID): The unique identifier of the agent.
+
+        Returns:
+            :obj:`dict`:
+                The agent dictionary record.
+
+        Examples:
+            >>> agent = tio.v3.vm.agents.details(
+            ...     '00000000-0000-0000-0000-000000000000'
+            ... )
+            >>> pprint(agent)
+        '''
+        return self._get(f'{agent_id}')
+
+    def list_agents_from_group(self, agent_group_id: UUID) -> Dict:
```

Comment:
This method also has pagination. Though, I think we still need to decide if this requires search or not. So, let's comment the code, and keep this as NotImplemented.